README.md

```diff
@@ -212,6 +212,107 @@ You may need to set up multiple automated transfer instances, for example if req
 
 In case different hooks are required for each instance, a possible approach is to checkout a new instance of the automation tools, for example in `/usr/lib/archivematica/automation-tools-2`
 
+Archivematica Client
+--------------------
```

Comment:
Can you add this section to the TOC at the top of the doc? Something like `- [Archivematica Client](#archivematica-client)`